frontend/src/Metamaps/Control.js

```diff
@@ -20,6 +20,21 @@ const Control = {
     node.setData('dim', 30, 'current')
     Selected.Nodes.push(node)
   },
+  selectNeighbors: function() {
+    if(Selected.Nodes.length > 0){
```

Comment:
Can you run

./node_modules/.bin/eslint --fix frontend/src/Metamaps

I think itll clean up a few style issues